src/design-system/typography/typeHierarchy.ts

```diff
@@ -36,6 +36,15 @@ export const typeHierarchy = {
         ios: -0.3,
       },
     },
+    '28px': {
```

Comment:
These new heading sizes (28 and 34) need go to in `Heading.examples.tsx` and `typeHierarchy.docs.tsx`. It might be worth automating those (and `Text.examples.tsx`) so new sizes don't get missed.